presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -848,10 +848,10 @@ public synchronized void revokeRoles(Set<String> roles, Set<HivePrincipal> grant
 
     private String getTableOwner(HiveIdentity identity, String databaseName, String tableName)
     {
-        return getSourceTableOrElseThrow(identity, databaseName, tableName).getOwner();
+        return getSourceTable(identity, databaseName, tableName).getOwner();
     }
 
-    private Table getSourceTableOrElseThrow(HiveIdentity identity, String databaseName, String tableName)
+    private Table getSourceTable(HiveIdentity identity, String databaseName, String tableName)
```

Comment:
Alternatives

- `getTable` -- it's already part of the class API and returns Optional
- `doGetTable` (what i proposed earlier) -- it would be applicable if called from `getTable`, but it's called in other places too
- `getSourceTable*` -- there is no "source" / "target" table in play, it's not applicable

taking this into account I back out and propose `getTableOrThrow` (sigh..)

any other options?

Reply:
renamed to `getExistingTableOrElseThrow`. What do you think?

Reply:
clever.
 why not just `getExistingTable`?

Reply:
> why not just getExistingTable?

done